### PR TITLE
update docs for the Ledger Post customization group

### PR DESCRIPTION
**Pull request by simonmichael**
Update to match the current four variables' names and help.

### CODE REVIEW

File: doc/ledger-mode.texi [hunk 1]
```diff
@@ -1146,11 +1146,21 @@ Ledger Post:
...
 
+@item ledger-post-account-alignment-column
+The column Ledger-mode attempts to align accounts to.
+
+@item ledger-post-amount-alignment-at
+Position at which the amount is aligned.
+
+Can be :end to align on the last number of the amount (can be
+followed by unaligned commodity) or :decimal to align at the
+decimal separator.
```
Comment (basil-conto) [inline doc/ledger-mode.texi:1157]: Should `:end` and `:decimal` be marked as `@code{}`?

Comment (simonmichael) [reply doc/ledger-mode.texi]: Thanks, I've re-pushed with that change.

**Comment by purcell** [comment]
Thanks!

**Comment by simonmichael** [comment]
@purcell: just a reminder that https://www.ledger-cli.org/3.0/doc/ledger-mode.html#Ledger-Post-Customization-Group needs to be re-rendered.

**Pull request merged**